source/simulator/stokes_matrix_free.cc

```diff
@@ -410,8 +410,8 @@ namespace aspect
 
         // Store the symmetric gradients of the velocity field and the
         // values of the pressure field
-        std::vector<SymmetricTensor<2,dim,VectorizedArray<number>>> sym_grad_u;
-        std::vector<VectorizedArray<number>> val_p;
+        AlignedVector<SymmetricTensor<2,dim,VectorizedArray<number>>> sym_grad_u;
+        AlignedVector<VectorizedArray<number>> val_p;
```

Comment:
This isn't wrong, but `std::vector` should (and as I far as I know from my experience playing with alignments in deal.II lately) honor alignment. What version of deal.II and what compiler was this with?

Reply:
From reading https://stackoverflow.com/a/8456491 it sounds like std::vector alignment is not enough for AVX.

Reply:
I didnt know about https://stackoverflow.com/a/8456491, all I saw was the explanation here: https://github.com/dealii/dealii/blob/64994cceda58b86f3dbf8b1a19eefa89064db41c/examples/step-37/step-37.cc#L203.